### PR TITLE
Fix: Temporary files created by the test are not removed.

### PR DESCRIPTION
**Pull request by liuxingbaoyu**
| Q                        | A 
| ------------------------ | ---
| Fixed Issues?            |  Temporary files created by the test are not removed.
| Patch: Bug Fix?          |√
| Major: Breaking Change?  |×
| Minor: New Feature?      |×
| Tests Added + Pass?      | ×
| Documentation PR Link    | 
| Any Dependency Changes?  |
| License                  | MIT

<a href="https://gitpod.io/#https://github.com/babel/babel/pull/14573"></a>

### CODE REVIEW

File: packages/babel-core/test/config-chain.js [hunk 1]
```diff
@@ -68,14 +68,41 @@ function escapeRegExp(string) {
...
   const tmp = name => path.join(cwd, name);
   const config = name =>
     pfs.copyFile(fixture("config-files-templates", name), tmp(name));
   return { cwd, tmp, config };
 }
 
+afterAll(() => {
+  function deleteDir(path) {
+    // Copy from packages\babel-cli\src\babel\util.ts
+    if (fs.existsSync(path)) {
```
Comment (nicolo-ribaudo) [inline packages/babel-core/test/config-chain.js:86]: We can add `rimraf` as a devDependency of `babel-core` (we already use it in other places), and just do `rimraf.sync(path)`. When we'll drop support for Node.js < 14 we can then do `fs.rmSync(path)`.

**Comment by nicolo-ribaudo** [comment]
(Off topic: I see two liuxingbaoyu accounts in our Slack. One with a green predefined picture and one with a red one. Which is your account?)

**Comment by liuxingbaoyu** [comment]
Oh no, I seem to have registered two accounts (because I rarely use slack and forgot my account😂).
Now I am logging into the red account.

**Review by nicolo-ribaudo — APPROVED** [review]

**Comment by nicolo-ribaudo** [comment]
The prettier e2e test is also failing on `main` (@sosukesuzuki do you know what might be causing that failure?)

**Pull request merged**